subscriber.rb

```diff
@@ -0,0 +1,49 @@
+require 'aws-sdk-sqs'
+require_relative 'load_aws'
+
+class Subscriber
+
+  attr_reader :sqs
+
+  def initialize
+    creds = Aws::Credentials.new(AWS["access_key_id"], AWS["secret_access_key"])
+    @sqs = Aws::SQS::Client.new(region: AWS["region"], credentials: creds)
+  end
+
+  def valid_sleep_period?(sleep_period)
+    max_sleep_period = /\A([3-9]|1[0-5])\z/
+    !sleep_period.nil? && sleep_period.match(max_sleep_period)
```

Comment:
Set a default value for `sleep_period` in the method definition. Make sure it is an invalid value and then you can get rid of the condition prior to the ampersand.